src/Migrations/Version20190408152238.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190408152238 extends AbstractMigration
```

Comment:
You should avoid to send two migrations if one was a mistake. Go back with `console doctrine:migration:migrate -n 20190405205103`, remove the files and start over.

Remember to remove `$this->addSql('CREATE SCHEMA public');` lines before do this.